### DISCUSSION TITLE
Default "English Tagged Messages" Profile Question

### DISCUSSION THREAD

**Discussion by blaxter75** [RomainLK/ytc-filter#54] [Q&A]
I don't know enough about coding or regex syntax to understand what the default "English Tagged Messages" catches. It appears to function properly. However, it caught a stray random message to display as a translated message. Someone entered in the phrase:

"T-took a b-bath??!"

I copy and pasted the message into chat to verify and ytc also filtered out my message to display and I'm not entirely sure what about this message is getting caught by the filter.

**Comment by RomainLK** [comment]
Oh... Github added this discussion feature.

It's a bug with the regex, which will be fixed in the next version. In the mean time, here is the fixed one:

`/^[[(]?(?:eng?|t(?:(rans|l))+|英訳)(?:\/(?:eng?|t(?:(rans|l))?|英訳))?[\]): -]/i`

The release will take some time however, because Google seems to be on holiday so review process is extremely slow, and I would rather not release during such period.

**Reply by blaxter75** [reply]
Thank you! I didn't want to throw it on issues since I wasn't sure if it was just me being incompetent
